tutorials/flume-capture-events.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: page
+title: Capturing Events with Flume
+---
+
+## Purpose
```

Comment:
Somewhere in this section, please note that this is optional and the reader can generate data with the other tutorial. That way, readers that aren't interested in Flume can skip this.